assets/global.js

```diff
@@ -580,6 +580,38 @@ class ModalDialog extends HTMLElement {
 }
 customElements.define('modal-dialog', ModalDialog);
 
+class BulkModal extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  connectedCallback() {
+    const handleIntersection = (entries, observer) => {
+      if (!entries[0].isIntersecting) return;
+      observer.unobserve(this);
+      if (this.innerHTML.trim() === '') {
+        const searchParam = this.dataset.url.includes('?') ? '&' : '?';
+        fetch(`${this.dataset.url}${searchParam}section_id=bulk-quick-order-list`)
```

Comment:
```suggestion
        const productUrl = this.dataset.url.split('?')[0];
        fetch(`${productUrl}?section_id=bulk-quick-order-list`)
```
On second thought, you don't need or want any other information except for the relative product URL. So this will only take part of the URL you want

Reply:
Ahhh this is a great idea 👍 thank you!